core/server/src/main/java/alluxio/proxy/FileSystemClientRestServiceHandler.java

```diff
@@ -89,407 +98,375 @@ public FileSystemClientRestServiceHandler(@Context ServletContext context) {
   }
 
   /**
-   * @summary get the service name
+   * @summary close a stream
+   * @param id the stream id
    * @return the response object
    */
-  @GET
-  @Path(SERVICE_NAME)
-  @ReturnType("java.lang.String")
-  public Response getServiceName() {
-    return RestUtils.call(new RestUtils.RestCallable<String>() {
-      @Override
-      public String call() throws Exception {
-        return Constants.FILE_SYSTEM_CLIENT_SERVICE_NAME;
-      }
-    });
-  }
-
-  /**
-   * @summary get the service version
-   * @return the response object
-   */
-  @GET
-  @Path(SERVICE_VERSION)
-  @ReturnType("java.lang.Long")
-  public Response getServiceVersion() {
-    return RestUtils.call(new RestUtils.RestCallable<Long>() {
+  @POST
+  @Path(STREAMS_PREFIX + ID_PARAM + CLOSE)
```

Comment:
this is sort of fragile since it assumes that each of these strings have compatible "/" startings/endings. For example, we couldn't use both path and id params with `STREAMS_PREFIX + PATH_PARAM + ID_PARAM + CLOSE`. I think it would be less fragile if we just wrote out the string, e.g. "streams/{id}/close".